sdks/python/apache_beam/__init__.py

```diff
@@ -65,17 +65,16 @@
 
 import sys
 
-
 if sys.version_info.major != 2:
   raise RuntimeError(
-      'Dataflow SDK for Python is supported only on Python 2.7. '
+      'Apache Bean SDK for Python is supported only on Python 2.7. '
       'It is not supported on Python [%s].' % sys.version)
 
-
-import apache_beam.internal.pickler
+__version__ = "0.2.0-incubating.dev"  # FIXME: PEP 440
```

Comment:
Can you please add a comment about why you decided this instead of something else? 
So that people know how to bump it up? Also this line is basically rewritten by code in setup.py. It is worth a comment.